scripts/deploy.sh

```diff
@@ -103,17 +112,16 @@ if [[ "${DEPLOY_AIO}" == "yes" ]]; then
       # In production, the OSDs will run on bare metal however in the AIO we'll put them in containers
       # so the MONs think we have 3 OSDs on different hosts.
       sed -i 's/is_metal: true/is_metal: false/' /etc/openstack_deploy/env.d/ceph.yml
-
       sed -i "s/journal_size:.*/journal_size: 1024/" $RPCD_VARS
-      echo "monitor_interface: eth1" | tee -a $RPCD_VARS
-      echo "public_network: 172.29.236.0/22" | tee -a $RPCD_VARS
       sed -i "s/raw_multi_journal:.*/raw_multi_journal: false/" $RPCD_VARS
-      echo "osd_directory: true" | tee -a $RPCD_VARS
-      echo "osd_directories:" | tee -a $RPCD_VARS
-      echo "  - /var/lib/ceph/osd/mydir1" | tee -a $RPCD_VARS
       sed -i "s/glance_default_store:.*/glance_default_store: rbd/" /etc/openstack_deploy/user_variables.yml
-      echo "nova_libvirt_images_rbd_pool: vms" | tee -a /etc/openstack_deploy/user_variables.yml
-      echo "cinder_ceph_client_uuid:"  | tee -a /etc/openstack_deploy/user_secrets.yml
+      add_config ceph
+    else
+      if [[ "$DEPLOY_SWIFT" == "yes" ]]; then
+        echo "glance_default_store: swift" | tee -a /etc/openstack_deploy/user_osa_variables_defaults.yml
```

Comment:
If it does do duplicate values, that issue will be short lived and has minimal effect. I say 'short lived' because the duplicate executions will be removed from gating once this merges across all branches, and I say 'minimal effect' because the values will be the same and therefore it does no harm.